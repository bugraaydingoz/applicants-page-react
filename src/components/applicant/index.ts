export { Applicant } from './Applicant'
export { ApplicantList } from './ApplicantList'
