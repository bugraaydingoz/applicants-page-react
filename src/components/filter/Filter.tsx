import React from 'react'

// Components
import { ReactComponent as DownIcon } from '../shared/icons/arrow-down.svg'

// Styles
import { FilterContainer } from './styles'
import { Body2 } from '../shared/typography'

interface Props {
  title: string
}

export function Filter({ title }: Props) {
  return (
    <FilterContainer>
      <Body2 style={{ marginRight: '4px' }}>{title}</Body2>
      <DownIcon />
    </FilterContainer>
  )
}
