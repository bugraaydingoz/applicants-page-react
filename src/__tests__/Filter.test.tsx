import React from 'react'
import renderer from 'react-test-renderer'
import { Filter } from '../components/filter'

test('renders correctly', () => {
  const tree = renderer.create(<Filter title="test" />).toJSON()
  expect(tree).toMatchSnapshot()
})
