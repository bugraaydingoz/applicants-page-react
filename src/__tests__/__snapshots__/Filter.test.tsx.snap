// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-AxjAm jDXiXI"
>
  <div
    className="sc-AxhUy hwVAoa"
    style={
      Object {
        "marginRight": "4px",
      }
    }
  >
    test
  </div>
  <svg>
    arrow-down.svg
  </svg>
</div>
`;
